content/id/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Contekan kubectl
+content_template: templates/concept
+card:
+  name: reference
+  weight: 30
+---
+
+{{% capture overview %}}
+
+Lihat juga: [Ikhitsar Kubectl](/docs/reference/kubectl/overview/) dan [Panduan JsonPath](/docs/reference/kubectl/jsonpath).
+
+Laman ini merupakan ikhitisar dari perintah `kubectl`.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+# kubectl - Contekan
+
+## Autocomplete Kubectl
+
+### BASH
+
+```bash
+source <(kubectl completion bash) # menyiapkan autocomplete untuk bash ke dalam shell saat ini, paket bash-completion harus diinstal terlebih dahulu.
+echo "source <(kubectl completion bash)" >> ~/.bashrc # menambahkan autocomplete secara permanen ke dalam bash shell kamu.
+```
+
+Kamu juga dapat menggunakan alias singkatan untuk `kubectl` yang juga bisa berfungsi dengan _completion_:
+
+```bash
+alias k=kubectl
+complete -F __start_kubectl k
+```
+
+### ZSH
+
+
+```bash
+source <(kubectl completion zsh)  # menyiapkan autocomplete untuk zsh ke dalam shell saat ini.
+echo "[[ $commands[kubectl] ]] && source <(kubectl completion zsh)" >> ~/.zshrc # menambahkan autocomplete secara permanen ke dalam zsh shell kamu.
+```
+
+## Konteks Kubectl dan Konfigurasinya
+
+Memilih klaster Kubernetes yang mana yang ditembak oleh `kubectl` untuk berkomunikasi dan
```

Comment:
I forgot whether we;re using klaster or kluster right now

Reply:
we are now using klaster, because the meaning is very close to what we want  https://kbbi.kemdikbud.go.id/entri/klaster
```
beberapa benda atau hal yang berkelompok menjadi satu; gugus
```